stage0/run.go

```diff
@@ -652,6 +684,10 @@ func setupStage1Image(cfg RunConfig, cdir string, useOverlay bool) error {
 		}
 	}
 
+	if err := setJournalPermissions(cfg, cdir); err != nil {
```

Comment:
hang on, this is coupling to stage1 implementation now right? Why can't we do it inside stage1?